hw/xbox/xbox.c

```diff
@@ -476,6 +476,34 @@ static void machine_set_eeprom(Object *obj, const char *value,
     ms->eeprom = g_strdup(value);
 }
 
+static char *machine_get_avpack(Object *obj, Error **errp)
+{
+    XboxMachineState *ms = XBOX_MACHINE(obj);
+
+    return g_strdup(ms->avpack);
+}
+
+static void machine_set_avpack(Object *obj, const char *value,
+                               Error **errp)
+{
+    XboxMachineState *ms = XBOX_MACHINE(obj);
+
+    if (strlen(value) == 0) {
+        /* pretend to have a composite av pack plugged in by default */
+        g_free(ms->avpack);
+        ms->avpack = g_strdup("composite");
+        return;
+    }
```

Comment:
Leaving marker for @mborgerson : dupliaction of empty-check and hardcoded default is here.

*(Also duplication of the code that's at the end of this function - avoidable by modifying `value`*)